tb_db/models.py

```diff
@@ -39,32 +39,32 @@ def __tablename__(cls):
 Base = declarative_base(cls=Base)
 
 
-@declarative_mixin
-class HistoryMixin:
-    created_at = Column(DateTime, default=func.now())
-    valid_until = Column(DateTime, default=None)
-
-    def new_version(self, session):
-        # invalidate current version - generate an update
-        old_id = self.id
-        session.query(self.__class__).filter_by(id=old_id).update(
-            values=dict(valid_until=func.now()), synchronize_session=False
-        )
-
-        # make us transient (removes persistent identity)
-        # make this an INSERT
-        make_transient(self)
-        self.id = None
-
-        # set created_at and valid_until, which means we have a new PK.
-        self.created_at = func.now()
-        self.valid_until = None
+#@declarative_mixin
```

Comment:
Could you add a small note here with the date, and a brief explanation of why this is here, and why it was commented out?